src/menus/shipSelectionScreen.cpp

```diff
@@ -298,7 +298,7 @@ ShipSelectionScreen::ShipSelectionScreen()
                 LOG(INFO) << "Password doesn't match control code. Attempt: " << password;
                 my_player_info->commandSetShipId(-1);
                 // Notify the player.
-                password_label->setText("Incorrect control code. Re-enter code for " + ship->getCallSign() + ":");
+                password_label->setText(tr("Incorrect control code. Re-enter code for ") + ship->getCallSign() + ":");
```

Comment:
Just like in the other PR, this needs a `.format` to remove the `+` and make it a single proper sentence to translate.